src/components/BookCard.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ]  As suggested by the previous reviewer, you shouldn't make use of disable eslint, rather pls fix your errors. I noticed that BookCard.js is currently generating a react/prop-types warning. To fix this, I suggest adding propTypes to your BookCard component. This will help validate the props passed to the component and prevent runtime errors. Here's an example of how you could add propTypes:

BookCard.propTypes = {
  bookProp: PropTypes.shape({
    id: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    author: PropTypes.string.isRequired,
  }).isRequired,
  delBook: PropTypes.func.isRequired,
};